command/runner.go

```diff
@@ -61,6 +67,20 @@ func Run(args []string, version string) error {
 
 	shouldColorize, subcommandInfo := ResolveSubcommand(args, config)
 
+	subcmd := subcommandInfo.Subcommand
+	if config.Pager && isOutputTerminal() &&
+		(subcmd == kubectl.Get || subcmd == kubectl.Describe ||
+			subcmd == kubectl.Logs || subcmd == kubectl.Explain) {
```

Comment:
Could we move this into the SubcommandInfo type somehow? So we just do a `subcommandInfo.SupportsPager()` or something like that?

The implementation of `SupportsPager` could just be a function with only a switch statement in it